examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -40,6 +43,16 @@ class GalleryDrawer extends StatelessWidget {
   final ValueChanged<bool> onShowPerformanceOverlayChanged;
 
   @override
+  _GalleryDrawerState createState() => new _GalleryDrawerState();
+}
+
+class _GalleryDrawerState extends State<GalleryDrawer> {
```

Comment:
Instead of making the drawer stateful, maybe separate out a stateful animated logo widget.